certification/pyxis/builder.go

```diff
@@ -129,17 +115,15 @@ func (b *certificationInputBuilder) WithArtifactFromFile(filepath string) *certi
 		Content:     base64.StdEncoding.EncodeToString(bytes),
 		ContentType: http.DetectContentType(bytes),
 		Filename:    path.Base(filepath),
-		FileSize:    info.Size(),
+		FileSize:    int64(len(bytes)),
```

Comment:
Perfect. I'm sure you tested as well, but documenting a self-contained test in case anyone else wants to try.

```
package main

import (
	"fmt"
	"os"
)

func main() {
	f := "file.txt"
	os.WriteFile(f, []byte("hello world\n"), 0755)

	file, _ := os.Open(f)
	info, _ := file.Stat()

	fileBytes, _ := os.ReadFile(f)
	fmt.Println("len bytes: ", int64(len(fileBytes)))
	fmt.Println("size from stat: ", info.Size())
}
```

Reply:
Happy to write a unit test for this.